RELEASE-NOTES-V2.md

```diff
@@ -0,0 +1,65 @@
+# Puik v2 (Beta) - Breaking Changes and Available Components
+
+## Breaking Changes
+
+In the transition from Puik v1 to Puik v2, we've made several significant changes to improve the library's performance, usability, and maintainability. Here are the key changes that you should be aware of:
+
+- **Monorepo Structure**: In Puik v2, we've transitioned to a monorepo structure. This structure includes separate libraries for Vue 3 components, Web components, a component resolver for our Vue 3 component library, a CSS library containing all the classes used in our components, a Tailwind CSS preset that contains all the design tokens used to create the components. This change was made to improve the modularity and maintainability of the project.
```

Comment:
"we moved to" instead "we've transitioned" + "and a Tailwind CSS " instead ", a Tailwind CSS"

Reply:
ok